HW1/P2/P2b.py

```diff
@@ -1,3 +1,57 @@
-from P2 import *
+# calling the file with all of the imports included
+execfile('P2.py')
 
-# Your code here
+import matplotlib
+import numpy as np
+import matplotlib.pyplot as plt
+import multiprocessing as mp
+import seaborn as sns
+sns.set_context('poster', font_scale=1.25)
+from random import randint
+
+
+# initializing spark
+import pyspark as ps
+
+config = ps.SparkConf()
+config = config.setMaster('local[' + str(2*mp.cpu_count()) + ']')
+config = config.setAppName('P2b')
+
+sc = ps.SparkContext(conf=config)
+
+#creating function for creating the pixels and passing on a tuple and the results
+def createPixels(row,column):
+    xx = (column/500.0)-2.0
+    yy = (row/500.0)-2.0
+    result = mandelbrot(xx,yy)
+    return (row,column),result
+
+# this creates an RDD that constains values in the range of 0 to 2000 stepping 
+# in increments of 1 each divided into 10 partitions so that 10*10=100 partitions
+row = sc.range(0,2000)
```

Comment:
you changed the code but not the comments. watch out for this - wrong comments are worse than no comments